frontend-ssr/pages/api/user/login.js

```diff
@@ -0,0 +1,74 @@
+//Imports
+const User = require("~/models/User.schema");
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+import dbConnect from "~/utils/dbConnect";
+const { loginValidation } = require("~/utils/validation/userValidation");
+
+//Connecting to DB
+dbConnect();
+
+//Vars
+const EMAIL_USERNAME_DOES_NOT_EXISTS_ERROR = {
+  status: "fail",
+  message: "Email or Username does not exists",
+};
+
+const BAD_FORM_SUBMITION_ERROR = {
+  status: "fail",
+  message: "Invalid form submision",
+};
+
+const BAD_PASSWORD_ERROR = {
+  status: "fail",
+  message: "Incorect password ",
+};
+
+const UPDATE_LAST_LOGIN_ERROR = {
+  status: "fail",
+  message: "Something went wrong. Please try again",
+};
+
+//Route function
+export default async (req, res) => {
+  let { method, body } = req;
+  switch (method) {
```

Comment:
instead of forcing a switch you can say if method != post =>  bad request and then you reduce nesting twice